.changeset/perfect-colts-scream.md

```diff
@@ -0,0 +1,6 @@
+---
+"@firebase/database-compat": patch
+"@firebase/database": patch
```

Comment:
It doesn't seem like the "@firebase/database" bump is needed.

Reply:
Removed